cmd/launcher/launcher.go

```diff
@@ -288,6 +289,22 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 		if err := controlService.RegisterConsumer(notificationconsumer.NotificationSubsystem, notificationConsumer); err != nil {
 			return fmt.Errorf("failed to register notify consumer: %w", err)
 		}
+
+		// Set up our tracing instrumentation
+		ingestConsumer := tokenconsumer.NewIngestTokenConsumer(k.TokenStore())
```

Comment:
if all this does is store tokens, maybe call it something like `tokenConsumer`, see comments on tokenconsumer

Reply:
discussed here https://github.com/kolide/launcher/pull/1226#discussion_r1230219621